packages/trader/src/Modules/SmartChart/Components/toolbar-widgets.tsx

```diff
@@ -1,9 +1,14 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { isDesktop, isMobile } from '@deriv/shared';
 import { ChartMode, DrawTools, Share, StudyLegend, Views, ToolbarWidget } from 'Modules/SmartChart';
 
-const ToolbarWidgets = ({ position, updateChartType, updateGranularity }) => {
+type TToolbarWidgets = {
+    position?: string;
+    updateChartType: (type: string) => void;
+    updateGranularity: (granularity: number) => void;
+};
+
+const ToolbarWidgets = ({ position, updateChartType, updateGranularity }: TToolbarWidgets) => {
```

Comment:
Do we need `position` as a prop?

Reply:
I think you are right. In files where `<ToolbarWidgets />` is imported the parent component doesn't pass such prop; in `<ToolbarWidgets />` itself if position is truthy or it's mobile, then we pass 'bottom' in ternary. 
I'll remove position, thank you for noticing